eslint.config.mjs

```diff
@@ -25,7 +25,6 @@ export default defineConfig([
 		// These are new love rules that we weren't following.  Disabling them here lets
 		// us re-enable them one-by-one alongside any necessary code changes.
 		rules: {
-			'@typescript-eslint/non-nullable-type-assertion-style': 'off',
```

Comment:
Looks like we must have fixed whatever this didn't like in the process of fixing the other rules before this one!